code/Model/EditableFormField/EditableFileField.php

```diff
@@ -194,7 +194,18 @@ public function validate()
         return $result;
     }
 
+    public function checkIsFolderExist()
```

Comment:
Could you please add a quick unit test for this new method

Reply:
I test this function as part of other test: https://github.com/silverstripe/silverstripe-userforms/blob/0a45f6a0dc5fed0293175ba9b0b63f06281da4de/tests/php/Model/EditableFormFieldTest.php#L180-L181
Do you want me to extract this tests to the new function?

Reply:
Yes, and it's OK if it's somewhat duplicated (don't spend too much time refactoring what you already have).  It should only take a few minutes to write a quick test here.